hig.js

```diff
@@ -87,17 +88,41 @@ function speak(phrases) {
 		return false;  	
 	}
 	var p = phrases.shift();
-	var v = 'Alex';
-	if (p.name == ANNOUNCER) {
-		v = 'Samantha';
-	} 
+	if(os == 'mac'){  //mac male voice to talk as hockey player
+		var v = 'Alex';
+			if (p.name == ANNOUNCER ) { //windows female voice to talk as announcer 
+				v = 'Samantha';
+		} 
+	}
+	// else if (os=='win32'){ //windows male voice to talk as hockey player
+		// var v = 'David';
+			// if (p.name == ANNOUNCER ) { //linux female voice to talk as announcer 
+				// v = 'Zira';
+		// }
+	// }
+	// else if (os == 'Darwin'){ // Linux Male voice to talk as hockey player
+		// var v = 'Linux_Male';
+			// if (p.name == ANNOUNCER ) { //linux female voice to talk as announcer
+				// v = 'Linux_Female';
+		// }
+	// }
 	var ph = p.phrase.toString().startsWith('(') ? "..." : p.phrase;
-	say.speak(ph, v, 1, (err) => {
-  		if (err) {
-    		return console.error(err);
-  		}
-  		speak(phrases);
-	});		
+	let p1 = new Promise(
+        // The executor function is called with the ability to resolve or
+        // reject the promise
+       (resolve, reject) => {
+			setTimeout( function() {
+				resolve(say.speak(ph, v, 1, (err) => {
+					if (err) {
+						return console.error(err);
+					}
+					setTimeout( function() {speak(phrases, os)}, 1000);
```

Comment:
Am I missing something the call here still appears to attempt to pass the os

Reply:
No, I did not change this but should have. My mistake, thanks.

Reply:
Fixed, once again I will make sure to double check.